docs/topics/ref_resolved-issues-7-2-0.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+//docs/release_notes-7.2.0/master.adoc
+
+:_template-generated: 2024-12-04
+:_mod-docs-content-type: REFERENCE
+
+[id="resolved-issues-7-2-0_{context}"]
+= Resolved issues
+
+This section provides highlighted issues that have been resolved in {ProductFullName} 7.2.0.
+
+.{ProductShortName} azure-appservice target is missing
+
+In previous versions of {ProductShortName}, the `azure-appservice` target was not available in the CLI distribution of {ProductFullName}. This issue has been resolved in the {ProductShortName} CLI. link:https://issues.redhat.com/browse/MTA-4036[(MTA-4036)]
+
+.Tags are not accessible in SVN repositories
+
+In previous versions of {ProductShortName}, when retrieving source code from Subversion (SVN) repositories, addons were automatically adding `/trunk` or `/branches/<target_branch>` to the repository URL, making it impossible to check out anything under tags. This issue has been resolved in the {ProductShortName} 7.2.0. link:https://issues.redhat.com/browse/MTA-4236[(MTA-4236)]
+
+.{ProductShortName} CLI: Binary analysis does not work on CLI as `pom.xml` is corrupted during extraction
+
+In previous versions of {ProductShortName}, the binary analysis did not function on the {ProductShortName} CLI, as the `pom.xml` was corrupted during extraction. This issue has been resolved now in the {ProductShortName} CLI. link:https://issues.redhat.com/browse/MTA-4019[(MTA-4019)]
+
+.Application import fails when Business Services exist already
+
+In previous versions of {ProductShortName}, importing applications using a CSV after a previous import that created Business Services led to the newly created applications having the incorrect Business Services assigned. This issue has been resolved in the {ProductShortName} 7.2.0. link:https://issues.redhat.com/browse/MTA-4257[(MTA-4257)]
+
+.{ProductShortName} CLI fails to handle `--bulk` properly on Windows platform
+
+In previous versions of {ProductShortName}, the bulk analysis on Windows systems could not be performed. This issue has been resolved now in the {ProductShortName} CLI. link:https://issues.redhat.com/browse/MTA-3624[(MTA-3624)]
+
+.IntelliJ IDEA cloud target is not functioning as expected
```

Comment:
Please rename this to "IntillJ IDE target list showed unusable targets"

This was actually impacting other issues too would be good to call out

Reply:
@dymurray - please can you send me a list of other issues and i will add as bullet points underneath:

```
This impacts other issues, including:
* 
```